src/tools/illink/test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpAnalyzerVerifier`1.cs

```diff
@@ -39,9 +39,14 @@ public static DiagnosticResult Diagnostic (DiagnosticId diagnosticId)
 			=> CSharpAnalyzerVerifier<TAnalyzer, XUnitVerifier>.Diagnostic (DiagnosticDescriptors.GetDiagnosticDescriptor (diagnosticId));
 
 		/// <inheritdoc cref="AnalyzerVerifier{TAnalyzer, TTest, TVerifier}.VerifyAnalyzerAsync(string, DiagnosticResult[])"/>
-		public static async Task VerifyAnalyzerAsync (string src, (string, string)[]? analyzerOptions = null, IEnumerable<MetadataReference>? additionalReferences = null, params DiagnosticResult[] expected)
+		public static async Task VerifyAnalyzerAsync (
+			string src,
+			bool consoleApplication,
```

Comment:
Nit: Make it `false` by default to avoid changing so many lines - and also to simplify the most common usage.